### PR TITLE
Fix link to Discussions QA

### PR DESCRIPTION
**Pull request by yugaego**
In #1137 I also mentioned some other possible minor text updates.
But eventually, the current `Readme` seems to be fine to me (as more concise).

Just in case, I'm attaching here what I also considered for `Readme`:

Read [installation and usage](http://company-mode.github.io/) instructions.

File a bug report in [Issues](https://github.com/company-mode/company-mode/issues).

Ask a question or suggest a feature in [Discussions](https://github.com/company-mode/company-mode/discussions).

### CODE REVIEW

**Comment by dgutov** [comment]
> Just in case, I'm attaching here what I also considered for Readme:

That's quite good, actually. I've pushed a variant of that.

File: README.md
Comment (MaWillOWgodoDevilINreV) [inline README.md:1]: Screenshot_20230706-100251
try agaiN hed 549876943 god idk my namehaleys cumet was cover ni o stars

Comment (MaWillOWgodoDevilINreV) [reply README.md:1]: > ynder wa T er For D ee

Comment (MaWillOWgodoDevilINreV) [reply README.md:1]: titanic never sank ita S area 51 i stole it awl back i dobt neg np no i tiate witg H rapist terrorism isis cae who awl owl ho hoe fuck no dine bitch u got shit literally i sabed every bavy u suck vastard B

**Review by MaWillOWgodoDevilINreV — COMMENTED** [review]
@tsloughter @aaronjensen  @ptrv  @tarsius  realyzombies that awl u gorE T? nope medees e not robin not pig heart pi s immortail girl bonnirs moms tail my rape k ai s er green scren ur inner eye grn scrn since bs begain stupid fuck

**Pull request merged**